### DISCUSSION TITLE
IMU data frame

### DISCUSSION THREAD

**Discussion by simutisernestas** [bitcraze/discussions#125] [Q&A]
I'm trying to calculate AI-deck camera's pose in IMU frame and it's a bit confusing how is the IMU frame itself defined. Maybe a better way to phrase it: is the IMU data aligned with body frame or it's coming directly from sensor. I suppose this alignment would be useful for using the measurements for control in body frame. There are some alignments in source code:
https://github.com/bitcraze/crazyflie-firmware/blob/a150e91cb028ac25f05ae0f1895466d5436e0e21/src/hal/src/sensors_bmi088_bmp388.c#L877-L920
but it seems to deal only with rotation. Could you please explain how's IMU frame defined?

**Comment by tobbeanton** [comment]
Are you maybe looking for [this](https://www.bitcraze.io/documentation/system/platform/cf2-coordinate-system/)?

I guess it depends on what data you are referring to. There are some comments in [stabilizer_types.h](https://github.com/bitcraze/crazyflie-firmware/blob/2a282b575c2541d3f3b4552296952a6cc40bf5b5/src/modules/interface/stabilizer_types.h). Is this enough?
